crates/astria-core/src/protocol/transaction/v1/action/mod.rs

```diff
@@ -1032,12 +1035,19 @@ impl Protobuf for Ics20Withdrawal {
             bridge_address,
             use_compat_address,
         } = proto;
-        let amount = amount.ok_or(Ics20WithdrawalError::field_not_set("amount"))?;
+        let amount = u128::from(amount.ok_or(Ics20WithdrawalError::field_not_set("amount"))?);
         let return_address = Address::try_from_raw(
             &return_address.ok_or(Ics20WithdrawalError::field_not_set("return_address"))?,
         )
         .map_err(Ics20WithdrawalError::return_address)?;
 
+        if timeout_time == 0 {
+            return Err(Ics20WithdrawalError::invalid_timeout_time());
+        }
+        if amount == 0 {
```

Comment:
Do we even need to check this? Protobuf represents 0 values as empty, and when decoding this it should result in `None` I think

Reply:
I don't even think we should reject this. If somebody wants to send (and pay for) a transfer, why would we not permit it?

Reply:
Removed in 84b00ef